### DISCUSSION TITLE
File upload

### DISCUSSION THREAD

**Discussion by opaque01** [derhansen/sf_event_mgt#1076] [Ideas]
Hallo

is there a posibillity to add a file upload field to the registration fields so that the users can upload a file during registration.
The uploaded file (e.g. a PDF) should also be attached in the confirmed event registration email.

thx opaque

**Comment by derhansen** [comment]
There is no possibility out of the box for this in the extension, but you can extend the main extension and addd the upload functionality yourself. There are already functions to use attachments (from custom properties) in the main extension - see https://docs.typo3.org/p/derhansen/sf_event_mgt/7.0/en-us/ForAdministrators/EmailAttachments/Index.html?highlight=files

Note, that I will not add this feature to the main extension - even if someone provides a PR for this.
